Somewhat spurious error message logged from web gateway when base_url is not present in the config

The example config has no base_url under gateways/web which seems to cause the following log message:

```
2014-07-14T16:17:04+09:30 [ERROR] :: flapjack-web :: base_url must contain trailing '/', setting it to safe default (/)
```

That should be a WARN at most. 

But if base_url is not supplied in the configuration, and '/' is a safe default (is it?) then log it as INFO to inform the admin of the value flapjack is setting this config to

--- a/lib/flapjack/gateways/web.rb
+++ b/lib/flapjack/gateways/web.rb
@@ -68,12 +68,14 @@ module Flapjack
           end
 
           @base_url = @config['base_url']
-          if @base_url
-            @base_url = $1 if @base_url.match(/^(.+\/)$/)
-          else
-	          dummy_url = "/"
-            @logger.error "base_url must contain trailing '/', setting it to safe default (#{dummy_url})"
-            @base_url = dummy_url
+          unless @base_url
+            @logger.info "base_url is not configured, setting to '/'"
+            @base_url = '/'
+          end
+
+          unless @base_url.match(/^.*\/$/)
+            @logger.warn "base_url must end with a trailing '/', setting to '#{@base_url}/'"
+            @base_url = "#{@base_url}/"
           end
 
           # constants won't be exposed to eRb scope

